### DISCUSSION TITLE
Make FZF_TMUX_OPTS default behavior

### DISCUSSION THREAD

**Discussion by joshmedeski** [junegunn/fzf#3389] [Ideas]
When setting `FZF_TMUX_OPTS` all uses of fzf-tmux don't recognize the options set unless explicitly passed into the command:

```sh
fzf-tmux $FZF_TMUX_OPTS
```

Why is this? Can this be rewritten so anytime someone calls `fzf-tmux` the options are read and used?

```sh
fzf-tmux # automatically use FZF_TMUX_OPTS
```

**Comment by junegunn** [comment]
FZF_TMUX_OPTS is currently only used by shell extensions, i.e. key bindings and fuzzy completion. The existence of the variable tells the scripts to use fzf-tmux instead of starting fzf with the `--height` option. So no, the variable wasn't designed to be used by fzf-tmux.


Anyway, I think your suggestion makes sense, although we can ignore the possibility that there are users who don't want it to affect their fzf-tmux outside of the shell extension context.

**Reply by joshmedeski** [reply]
We could create a new `FZF_TMUX_DEFAULT_OPTS` variable that works similarly to `FZF_DEFAULT_OPTS`.

Then, we wouldn't have to worry about communicating a breaking change on the previous variable. 

For now, I'm going to do something similar to what your doing on the key-binding and fuzzy completion logic to [detect that variable manually](https://github.com/joshmedeski/t-smart-tmux-session-manager/pull/45).

**Reply by junegunn** [reply]
As an alternative, you can define a function of the same name like so:

```sh
fzf-tmux() {
  command fzf-tmux -p 80% "$@"
}
```

**Reply by joshmedeski** [reply]
Alright, it sounds like you'd prefer putting the responsibility on the user to use `FZF_DEFAULT_OPTS` or not. 

I'm okay with that and I'll be integrating `FZF_DEFAULT_OPTS` support into my script, thanks.

**Reply by stherold** [reply]
What about this:

```
if [ -n "${TMUX:-}" ]; then
    # We are in tmux, so run fzf-tmux in poopup mode
    alias fzf="fzf-tmux -p90%,70%"
else
    # We are not in tmux, so run fzf normally
    unalias fzf
fi
```
